target/generated-sources/annotations/com/epam/spring/homework3/mapper/DishMapperImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.spring.homework3.mapper;
+
+import com.epam.spring.homework3.dto.DishDTO;
+import com.epam.spring.homework3.dto.DishDTO.DishDTOBuilder;
+import com.epam.spring.homework3.model.Dish;
+import com.epam.spring.homework3.model.Dish.DishBuilder;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Generated;
+import org.springframework.stereotype.Component;
+
+@Generated(
+    value = "org.mapstruct.ap.MappingProcessor",
+    date = "2022-07-18T04:24:50+0300",
+    comments = "version: 1.4.2.Final, compiler: javac, environment: Java 15.0.2 (Oracle Corporation)"
+)
+@Component
+public class DishMapperImpl implements DishMapper {
+
+    @Override
+    public DishDTO mapDishDTO(Dish dish) {
+        if ( dish == null ) {
+            return null;
+        }
+
+        DishDTOBuilder dishDTO = DishDTO.builder();
+
+        dishDTO.id( dish.getId() );
+        dishDTO.name( dish.getName() );
+        dishDTO.calories( dish.getCalories() );
+        dishDTO.ingredients( dish.getIngredients() );
+        dishDTO.category( dish.getCategory() );
+
+        return dishDTO.build();
+    }
+
+    @Override
+    public Dish mapDish(DishDTO dishDTO) {
+        if ( dishDTO == null ) {
```

Comment:
You shouldn't use extra spaces between words and keywords, it leads to bad readability and it's not a java style convention.